src/pages/o/[orgId]/campaigns.tsx

```diff
@@ -45,7 +47,9 @@ const OrgCampaignsPage : PageWithLayout<OrgCampaignsPageProps> = (props) => {
 
     return (
         <>
-            <h1>Campaigns for { orgQuery.data?.title }</h1>
+            <Heading level={ 1 }>
+                <Msg id="pages.o.orgId.campaigns.title"/> { orgQuery.data?.title }
```

Comment:
Although the message ID here does reflect the URL path and the path of the source file, it's unnecessarily long (and contains a lot of meta-information). All we require from it is that it's unique. A better path would be `pages.orgCampaignsPage.…` (derived from the name of the component in this file).